app/src/main/java/fr/groggy/racecontrol/tv/f1tv/F1TvClient.kt

```diff
@@ -38,7 +37,8 @@ class F1TvClient @Inject constructor(
     private val archiveSortInstant = Instant.now()
 
     suspend fun getSeason(archive: Archive): F1TvSeason {
-        val response = get(LIST_SEASON.format(archive.year), seasonResponseJsonAdapter)
+        val listSeasonUrl = "/2.0/R/${localeManager.currentLocale}/BIG_SCREEN_HLS/ALL/PAGE/SEARCH/VOD/F1_TV_Pro_Monthly/$GROUP_ID?filter_objectSubtype=Meeting&filter_season=%s&filter_orderByFom=Y&maxResults=100"
+        val response = get(listSeasonUrl.format(archive.year), seasonResponseJsonAdapter)
```

Comment:
Instead of this string interpolation, rever these things to strings and use a `%s` placeholder so it is mode visible for us when we have to change something. So then it would be `LIST_SEASON.format(currentLocale, archive.year)` and so on.